manifests/base/cluster-health-monitor/deployment.yaml

```diff
@@ -23,7 +23,7 @@ spec:
           image: mcr.microsoft.com/aks/cluster-health-monitor/cluster-health-monitor:v0.0.17
           args:
             - "--v=3"
-          imagePullPolicy: IfNotPresent
```

Comment:
The change in this file can be undone. I have a separate PR for this one: https://github.com/Azure/cluster-health-monitor/pull/220. It also requires a work around because `ImagePullPolicy: Always` doesn't work for local (KIND) tests (reasons described in comments in linked PR). 

Also, probably a heads up that you might need to create a patch for the imagePullPolicy whenever you create the PR for the e2e that uses it.